addons/udes_stock/models/stock_move.py

```diff
@@ -530,3 +530,14 @@ def _action_assign(self):
         """
         super()._action_assign()
         return self
+
+    def _search_picking_for_assignation(self):
```

Comment:
Can you add a comment explaining why we need this change please :slightly_smiling_face: and for `_get_new_picking_values`

Reply:
Sorry the comment does not explain why we really need the changes, 
for `_search_picking_for_assignation` - `When searching for pickings to assign moves too, the picking should have the same priority as the sales order. `
for `_get_new_picking_values` - `When creating a picking from a move, the picking should have the same priority as sales order from which it is created.`